app/Http/Controllers/Api/CourseController.php

```diff
@@ -262,6 +262,32 @@ public function bulkCourseAssign(Request $request)
         }
     }
 
+    public function enrollStudent(Request $request)
+    {
+        $validator = Validator::make($request->all(), [
+            'course_id' => 'required',
+            'enrollment_code' => 'required',
```

Comment:
The enrollment code isn't required for IBBUL students, it's only needed for Baze University Students. You may want to remove it here.

Reply:
Yes sir, please should not be required, u can just enroll by user_id and course_id